test/functional/page_objects/settings_page.ts

```diff
@@ -1048,4 +1053,24 @@ export class SettingsPageObject extends FtrService {
       [data-test-subj="indexPatternOption-${newIndexPatternTitle}"]`
     );
   }
+
+  async openEditFlyoutByRowNumber(rowNumber: number = 0) {
+    const editBtns = await this.testSubjects.findAll('editFieldFormat');
+    const editFlyoutBtn = editBtns[rowNumber];
+    await editFlyoutBtn.click();
+  }
+
+  async assertEditFlyoutByFieldType(name: string, fieldType: string) {
```

Comment:
since it does action and then assert, maybe better to call it `changeAndValidateFieldFormat` ?

Reply:
https://github.com/elastic/kibana/pull/194738/commits/b0ec24fd269738a0de0a3d9363bd3224e4028cb6